docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -8,6 +8,102 @@ ifdef::notBuildingForSite[]
 include::{includedir}/menu.adoc[]
 endif::notBuildingForSite[]
 
+== Getting Ehcache
+
+The latest version of Ehcache 3.10.0 has been released to maven central under the following coordinates:
+
+=== From Maven
+
+Add the following to the `<dependencies>` section in your `pom.xml`:
+
+[source,xml]
+----
+<dependency>
+  <groupId>org.ehcache</groupId>
+  <artifactId>ehcache</artifactId>
+  <version>3.10.0</version>
+</dependency>
+----
+
+=== From Gradle
+
+Add the following to the dependencies section of your `build.gradle`:
+
+[source,groovy]
+----
+dependencies {
+  implementation 'org.ehcache:ehcache:3.10.0'
+}
+----
+
+=== JAXB Dependencies
+
+Ehcache uses JAXB to parse its XML configuration files. For out-of-the-box functionality it depends on the JAXB
+reference implementation using a version range: `org.glassfish.jaxb:jaxb-runtime:[2.2,3)`.
+If you require a repeatable build you will need to depend on an explicit version within this range.
+If you wish to use an alternate implementation of JAXB then you may need to manually exclude this dependency.
+
+NOTE: If you are using Java 8, or Java 9 (with `--add-modules java.xml.bind`) then you may safely exclude this
+dependency in favor of the implementation provided by the JRE.
+
+=== Jakarta Variants
+
+Ehcache 3.10 adds support for the Jakarta EE namespaced versions of the JAXB APIs.
+If you have moved to the new Jakarta EE namespaced APIs then you may want to choose to use the matching Ehcache variants.
+
+==== From Maven
+
+Add the following to the `<dependencies>` section in your `pom.xml`:
+
+[source,xml]
+----
+<dependency>
+  <groupId>org.ehcache</groupId>
+  <artifactId>ehcache</artifactId>
+  <version>3.10.0</version>
+  <classifier>jakarta</classifier>
+</dependency>
+----
+
+[WARNING]
+====
+The Maven POM cannot capture the dependencies of the Jakarta variant.
```

Comment:
"capture"?  Does that mean "express" or "describe"?  


(While the dependencies of the Jakarta variant of Ehcache can be described using Gradle metadata, there is no corresponding description using a Maven POM.  When using the Jakarta Ehcache variant in a Maven project, you will also need ... )


You might consider just including the following dependencies section along with the Jakarta dependency and with a possible footnote as to why the extras are needed.

Reply:
Fixed, and then discussed.